src/DynamoCore/Core/CustomNodeDefinition.cs

```diff
@@ -274,12 +295,39 @@ public CustomNodeInfo(Guid functionId, string name, string category, string desc
                 Category = Dynamo.Properties.Resources.DefaultCustomNodeCategory;
         }
 
+        /// <summary>
+        /// Custom node unique ID
+        /// </summary>
         public Guid FunctionId { get; set; }
+
+        /// <summary>
+        /// Custom node name
+        /// </summary>
         public string Name { get; set; }
+
+        /// <summary>
+        /// Custom node category
+        /// </summary>
         public string Category { get; set; }
+
+        /// <summary>
+        /// Custom node description
+        /// </summary>
         public string Description { get; set; }
+
+        /// <summary>
+        /// Path to custom node
+        /// </summary>
         public string Path { get; set; }
+
+        /// <summary>
+        /// Bool value, that indicates if custom node is part of the package or not.
+        /// </summary>
         public bool IsPackageMember { get; set; }
+
+        /// <summary>
+        /// Bool value, that sets visibility in library search
+        /// </summary>
```

Comment:
Indicates if custom node is part of the library search..
If true, then custom node is part of  library search.